doc: changelog fixes (#10770)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

nRF9160: Asset Tracker v2
-------------------------

* Added the integration of the :ref:`lib_lwm2m_client_utils` FOTA callback functionality.

* Updated:

  * The application now uses the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.

>   * The application now uses defines from the :ref:`lib_nrf_cloud` library for nRF Cloud-related string values.

Author: I think second option is better. I have updated in all the three places.

>   * The application to use the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.
>   * The application to use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud-related string values.

Reviewer: ```suggestion
  * The application now uses the function :c:func:`nrf_cloud_location_request_msg_json_encode` to create an nRF Cloud location request message.
  * The application now uses defines from the :ref:`lib_nrf_cloud` library for nRF Cloud-related string values.
```

nRF9160: Serial LTE modem
-------------------------

* Added AT command ``#XWIFIPOS`` to get Wi-Fi location from nRF Cloud.
* Updated the application to use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.

> * Fixed a bug in receiving large MQTT Publish message.

Reviewer 2: ```suggestion
* Fixed a bug in receiving a large MQTT Publish message.
```

Added:

> * The :ref:`nrf_desktop_swift_pair_app` module.

Reviewer: Module name already has 'module'.
```suggestion
* The :ref:`nrf_desktop_swift_pair_app` .
```

* :ref:`modem_shell_application` sample:

  * Updated the sample to use defines from the :ref:`lib_nrf_cloud` library for nRF Cloud related string values.

>     Remove the inclusion of the file :file:`nrf_cloud_codec.h`.

Reviewer: ```suggestion
    Removed the inclusion of the file :file:`nrf_cloud_codec.h`.
```

* :ref:`https_client` sample:

>   * Added IPv6 support and wait for PDN to fully activate (including IPv6, if available) before looking up the address.

Reviewer 2: This sounds a bit weird. "Added IPv6 support and wait for..."
However, I don't have a good suggestion to make this better.

Author: I updated the sentence. Please check.
